src/components/VideoCard/VideoCard.spec.jsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+
+import VideoCard from './VideoCard.component';
+import ThemeProvider from '../../providers/Theme';
+import DarkModeProvider from '../../providers/DarkMode';
+
+describe('Testing VideoCard.component', () => {
+  const mockedGoToVideo = () => {};
```

Comment:
You can test that this was actually clicked if you wrap it using `jest.fn` like so:

```javascript
const mockedGoToVideo = jest.fn(() => {});
```

with this you'll have a function that is actually working, but it can be asserted using jest helpers like `toHaveBeenCalledTimes` and so on.